src/styles/Detailspage.css

```diff
@@ -0,0 +1,110 @@
+.detailspage {
+  font-family: Arial, sans-serif;
+}
+
+.parent {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+  width: 100%;
+  margin-top: 30px;
+}
+
+.items {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: rgba(54, 37, 114, 0.518);
+  border-radius: 10px;
+  text-align: center;
+}
+
+.image {
+  border-radius: 10px;
+  margin-top: 13px;
+  padding: 1px;
+  width: 60px;
+}
+
+.symbol {
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.company-name {
+  font-size: 24px;
```

Comment:
- [ ] Please apply the same layout in the given design. Your image and company name are one under the other. But in the actual design, you need your logo to be on the left and your name on the right side.
![image](https://github.com/waelelsafty07/Metrics-webapp/assets/91486791/9b90d341-c49c-49a2-8f55-a41a5be49443)
![image](https://github.com/waelelsafty07/Metrics-webapp/assets/91486791/70c2fdcb-76c9-4ecc-bd93-26602e338c71)